include/mqtt/endpoint.hpp

```diff
@@ -4191,10 +4191,10 @@ class endpoint : public std::enable_shared_from_this<endpoint<Mutex, LockGuard,
                         case control_packet_type::pingresp:
                         case control_packet_type::disconnect:
                             return remaining_length_ == 0;
-                        default:
+                        case control_packet_type::auth:
                             return false;
                         }
-                        break;
+                        return false;
```

Comment:
*1 I think that original code `break` is redundant because default covers it. Similarly `return false;` is also redundant. I think that there is no essential difference.
I prefer to the original code.

Reply:
Or simply remove `break;`. Does this removal make warning?

Reply:
If we don't have return false; after the auth case, we would get a warning about unintentional fallthrough.

When mqtt_cpp uses C++17, we can add [[fallthrough]] attribute to suppress warning.